sos/report/plugins/juju.py

```diff
@@ -121,10 +165,11 @@ def postproc(self):
         ]
 
         # Redact simple yaml style "key: value".
-        keys_regex = fr"(^\s*({'|'.join(protect_keys)})\s*:\s*)(.*)"
+        keys_regex = rf"(^\s*({'|'.join(protect_keys)})\s*:\s*)(.*)"
         sub_regex = r"\1*********"
         self.do_path_regex_sub(agents_path, keys_regex, sub_regex)
         # Redact certificates
         self.do_file_private_sub(agents_path)
 
+
 # vim: set et ts=4 sw=4 :
```

Comment:
I think you need to add a new line after the vim modeline to avoid a linting error.